solidity/contracts/bridge/BitcoinTx.sol

```diff
@@ -115,9 +115,10 @@ library BitcoinTx {
 
     /// @notice Represents info about an unspent transaction output.
     struct UTXO {
-        /// @notice Hash of the transaction the output belongs to.
+        /// @notice Hash of the transaction (little-endian) the output
+        ///         belongs to.
```

Comment:
I am confused about this change. `bytes32` is a byte array and it does not have endianness, array elements are always read in the same order. We can talk about endianness when we convert bytes to integers but I do not see us converting `txHash` to integer in this PR.

See that all references to endianness in this file talk about unsigned int encoding. For example:
https://github.com/keep-network/tbtc-v2/blob/3fdd8de8ed2787db028490c5d3b5467f3389cd5a/solidity/contracts/bridge/BitcoinTx.sol#L83-L84

https://github.com/keep-network/tbtc-v2/blob/3fdd8de8ed2787db028490c5d3b5467f3389cd5a/solidity/contracts/bridge/BitcoinTx.sol#L100-L101

Reply:
Yeah but look at my other comment: https://github.com/keep-network/tbtc-v2/pull/128#discussion_r806696905. I think it would be good to make it explicit. Mixing the byte order when using tx hash in any way can lead to serious problems.

Reply:
I reworded it to avoid saying "little-endian" while still flagging it should be the same as Bitcoin internal representation. See dbb7bf37